src/Compilers/Core/Portable/Operations/IInstanceReferenceExpression.cs

```diff
@@ -13,11 +13,6 @@ namespace Microsoft.CodeAnalysis.Semantics
     /// </remarks>
     public interface IInstanceReferenceExpression : IOperation
```

Comment:
Is this interface still needed?

Reply:
Yes. We need it as the receiver for accesses and invocations.

Reply:
Could the receiver be `IOperation` in those cases though?

Reply:
I'm not sure I understand? There is no base IOperation class in the IOperation tree.  All nodes in an actual IOperation tree are leaf nodes in the IOperation type hierarchy.

Reply:
The receiver instances could be `InstanceReferenceExpression` instances but that class would implement `IOperation` only.

Reply:
Right, but it's one of our design principles that IOperation trees are only constructed from leaf nodes, and that you'll never encounter something that's just an IOperation.